web-app-router/tailwind.config.ts

```diff
@@ -134,7 +135,7 @@ const config: Config = {
         32: "calc(32 * 1rem / 16)",
       },
 
-      width: {
+      spacing: {
```

Comment:
Why are we changing this?

Reply:
This is allows to use these values with other classes, for example, I was needed it with max-w today. It also applies to paddings, margins, etc. Just not to duplicate